src/main/java/sirius/biz/model/Constants.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.model;
+
+import sirius.kernel.Sirius;
+import sirius.kernel.commons.Strings;
+import sirius.kernel.nls.NLS;
+
+import java.util.List;
+
+/**
+ * Provides helpful constants and methods regarding language configuration data
+ * (and known currencies, countries, ... coming soon™)
+ */
+public class Constants {
+
+    private Constants() {
+    }
+
+    public static List<String> getLanguages() {
+        return Sirius.getSettings().getStringList("nls.languages");
```

Comment:
doesn't NLS provides this already? if not, why not add the method there?

Reply:
I looked at how language parameters are handled in Oxomi and that's how it's done there 🤷‍♂ The Constants Class holds a lot more constants and functions there though...